ara/static/css/ara.css

```diff
@@ -0,0 +1,34 @@
+.FAILED {
```

Comment:
The capitals here are silly but that's my fault. We should remove the capitalization and use {{ string|upper }} in jinja instead.